engine/flink/executor/src/test/scala/pl/touk/nussknacker/engine/process/compiler/StubbedFlinkProcessCompilerTest.scala

```diff
@@ -73,12 +73,12 @@ class StubbedFlinkProcessCompilerTest extends AnyFunSuite with Matchers {
 
   test("stubbing for test purpose should work for multiple sources") {
     val scenarioTestData = ScenarioTestData(List(
-      ScenarioTestRecord("left-source", Json.fromLong(11)),
-      ScenarioTestRecord("right-source", Json.fromLong(21)),
-      ScenarioTestRecord("right-source", Json.fromLong(22)),
-      ScenarioTestRecord("left-source", Json.fromLong(12)),
-      ScenarioTestRecord("left-source", Json.fromLong(13)),
-      ScenarioTestRecord("right-source", Json.fromLong(23)),
+      ScenarioTestJsonRecord("left-source", Json.fromLong(11)),
+      ScenarioTestJsonRecord("right-source", Json.fromLong(21)),
+      ScenarioTestJsonRecord("right-source", Json.fromLong(22)),
+      ScenarioTestJsonRecord("left-source", Json.fromLong(12)),
+      ScenarioTestJsonRecord("left-source", Json.fromLong(13)),
+      ScenarioTestJsonRecord("right-source", Json.fromLong(23)),
     ))
 
     val compiledProcess = testCompile(scenarioWithMultipleSources, scenarioTestData)
```

Comment:
Add a test for test-with-form for Flink engine here?

Reply:
I've added simple test case to show how it works :+1: